src/class/emitter.helper.ts

```diff
@@ -0,0 +1,29 @@
+import * as forge from 'node-forge';
+
+export class EmitterHelper {
+  public static getEmitterFromPfx(pfx: Buffer, passphrase: string): string {
+    let organizationalUnit;
+    try {
+      const p12Der = forge.util.decode64(pfx.toString('base64'));
+      const p12Asn1 = forge.asn1.fromDer(p12Der);
+      const p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, false, passphrase);
+      const bags = p12.getBags({ bagType: forge.pki.oids.certBag });
+      const certBag = bags[forge.pki.oids.certBag];
+      if (!certBag) { throw 'Unable to find certBag'; }
```

Comment:
Always throw an error to have the stacktrace attached

Reply:
this error will be catched and rethrown as an error with a stacktrace